drivers/wdm/audio/backpln/portcls/pool.cpp

```diff
@@ -8,31 +8,26 @@
 
 #include "private.hpp"
 
+#if 0
```

Comment:
Is that doing any speedup or what is the reason you did that? I don't oppose it, it is just not mentioned in the PRs description. Therefore the question.

Reply:
'debug.h' is useless in this file currently.

Reply:
Then let's strip that block entirely.

Reply:
I would rather postpone removal until I check the `YDEBUG` thing.
Nonetheless, I'll remove it if another dev wants that removal too.

Reply:
If you haven't checked the YDEBUG thing yet, and therefore are not sure about the consequences of removing the entire block, then you should definitely not place that #if 0 there! Because that does have the same impact as a complete removal.
So please either remove the entire block, or remove your added #if 0.

Reply:
> If you haven't checked the YDEBUG thing yet,

I have not, because it is a separate matter.

> and therefore are not sure about the consequences of removing the entire block, then you should definitely not place that #if 0 there! Because that does have the same impact as a complete removal.

Of course the two options have "identical" result.

> So please either remove the entire block, or remove your added #if 0.

Reverted.